pdns/syncres.hh

```diff
@@ -1012,7 +1012,9 @@ public:
   enum stateenum {BYTE0, BYTE1, GETQUESTION, DONE} state{BYTE0};
   uint16_t qlen{0};
   uint16_t bytesread{0};
-
+  uint16_t d_requestsInFlight{0}; // number of mthreads spawned for this connection
+  // The max number of concurrent TCP requests we're willing to process
+  uint16_t d_maxInFlight;
```

Comment:
I'm wondering if that could not be a static member, only updated once instead of reading `max-concurrent-requests-per-tcp-connection` for each incoming TCP connection?